### ISSUE TITLE
Hidden input for client state must have title per WCAG2

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2217]
We always generate a:

```xml
<input type="text" name="$client-state" value="" class="xforms-initially-hidden">
```

Even if this input is never shown, this is in violation of WCAG2 1.1.1/1.3.1/3.3.2 (A), reported as E866 by TotalValidator. Since this input is never shown we can just solve the problem by adding a `title=""` attribute.

**Comment by ebruchez** [comment]
And this cannot be an `<output>` since the value must be restored by the browser.

**Comment by ebruchez** [comment]
We don't need to support IE8 and IE9 anymore, so we could definitely make use of the history API (#1802).
